internal/arch/command-table.go

```diff
@@ -70,5 +72,9 @@ func (DBCommand) Execute(db *db.DB, cmd string, args []string) *protcl.Message {
 		return protcl.NewMessage(nil, err)
 	}
 
+	if argsLen := len(args); (command.MinArgs != -1 && argsLen < command.MinArgs) || (command.MaxArgs != -1 && argsLen > command.MaxArgs) {
```

Comment:
we can use 0 I guess, We can not have minimum infinite args, so its zero when it requires minimum 0 args. Sorry about introducing min as -1. Also `min(len) = 0` which i forgot 🗡